ee/session_recordings/ai/generate_embeddings.py

```diff
@@ -171,7 +171,10 @@ def generate_recording_embeddings(session_id: str, team: Team | int) -> List[flo
 
     client = OpenAI()
 
-    session_metadata = SessionReplayEvents().get_metadata(session_id=str(session_id), team=team)
+    eight_days_ago = datetime.datetime.now(pytz.UTC) - datetime.timedelta(days=8)
+    session_metadata = SessionReplayEvents().get_metadata(
+        session_id=str(session_id), team=team, recording_start_time=eight_days_ago
+    )
```

Comment:
```suggestion
    start_time = datetime.datetime.now(pytz.UTC) - datetime.timedelta(days=8)
    session_metadata = SessionReplayEvents().get_metadata(
        session_id=str(session_id), team=team, recording_start_time=start_time
    )
```

otherwise, we have to change 2 things if we decide to change from 8 to something else, if we don't forget to change the var name.
also, how did you come up with the 8 magic number? if there's a reason, worth adding a comment.

Reply:
Ah, fair usually.

It's not the start time in this case, just a floor for the query

I chose it because it's only slightly more than we're loading elsewhere

We're still playing with all of this so it'll change frequently - although many crimes have been committed after starting with that phrase 🤣

Reply:
lol, i renamed to start date because of `recording_start_time`, feel free to put what makes more sense.